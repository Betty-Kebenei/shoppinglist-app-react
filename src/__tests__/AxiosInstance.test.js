import React from 'react';
import { shallow, mount, render } from 'enzyme';

import Config from '../Config';

import AxiosInstance from '../components/AxiosInstance';

describe('interface of AxiosInstance component', () => {
    it('should render', () => {
        const wrapper = shallow(<AxiosInstance />);
        expect(wrapper).toMatchSnapshot();
    });
});