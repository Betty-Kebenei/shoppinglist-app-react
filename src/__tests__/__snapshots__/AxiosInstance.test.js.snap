// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`interface of AxiosInstance component should render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <wrap />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Promise {},
  Symbol(enzyme.__nodes__): Array [
    Promise {},
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
